wp-content/themes/inf_theme/admin/class-admin.php

```diff
@@ -18,49 +17,28 @@
  */
 class Admin extends Config {
 
-  /**
-   * General Helper class
-   *
-   * @var object General_Helper
-   *
-   * @since 2.0.1
-   */
-  public $general_helper;
-
-  /**
-   * Initialize class
-   *
-   * @param Helpers\General_Helper $general_helper Helper class instance.
-   *
-   * @since 3.0.0 Removing theme name and version.
-   * @since 2.0.0
-   */
-  public function __construct( General_Helper $general_helper ) {
-    $this->general_helper = $general_helper;
-  }
-
   /**
    * Register the Stylesheets for the admin area.
    *
-   * @since 2.0.0
+   * @since 1.0.0
    */
   public function enqueue_styles() {
 
-    $main_style = $this->general_helper->get_manifest_assets_data( 'applicationAdmin.css' );
-    wp_register_style( static::THEME_NAME . '-style', $main_style );
+    $main_style = General_Helper::get_manifest_assets_data( 'applicationAdmin.css' );
+    wp_register_style( static::THEME_NAME . '-style', $main_style, array(), static::THEME_VERSION );
     wp_enqueue_style( static::THEME_NAME . '-style' );
 
   }
 
   /**
    * Register the JavaScript for the admin area.
    *
-   * @since 2.0.0
+   * @since 1.0.0
    */
   public function enqueue_scripts() {
 
-    $main_script = $this->general_helper->get_manifest_assets_data( 'applicationAdmin.js' );
-    wp_register_script( static::THEME_NAME . '-scripts', $main_script );
+    $main_script = General_Helper::get_manifest_assets_data( 'applicationAdmin.js' );
+    wp_register_script( static::THEME_NAME . '-scripts', $main_script, array(), static::THEME_VERSION ); // phpcs:ignore WordPress.WP.EnqueuedResourceParameters.NotInFooter
```

Comment:
Why aren't we loading the scripts in the footer? I thought that it's better to load them all in the footer.

Reply:
because this is up to the developer to decide when to load scripts we can load them in the footer in the boilerplate.